solr/core/src/java/org/apache/solr/client/solrj/embedded/JettySolrRunner.java

```diff
@@ -697,12 +694,6 @@ public synchronized void stop() throws Exception {
         timeout.waitFor("Timeout waiting for reserved executor to stop.", rte::isStopped);
       }
 
-      // we want to shutdown outside of jetty cutting us off
-      SolrDispatchFilter sdf = getSolrDispatchFilter();
-      if (sdf != null) {
-        ExecutorUtil.shutdownAndAwaitTermination(getJettyShutDownThreadPool());
```

Comment:
This looks dubious (the code, not the removal) but I'm unclear if that's the same bug you're addressing here or a separate problem.

Reply:
It's unrelated, this executor isn't used for anything (at least that I can tell)